src/components/base/button/index.jsx

```diff
@@ -1,14 +1,16 @@
 import React from "react";
 import propTypes from "prop-types";
+import { connect } from "react-redux";
 import styles from "components/base/button/index.module.scss";
 import { TYPE_SUBMIT } from "utils/const";
 
-function Button({ children, onClick, className, buttonType }) {
+function Button({ children, onClick, className, buttonType, isDisabled }) {
```

Comment:
Could we be more explicit naming variables ?
```suggestion
function Button({ children, onClick, className, buttonType, isButtonDisabled }) {
```